src/bin/rustc-fake.rs

```diff
@@ -0,0 +1,60 @@
+use std::env;
+use std::process::Command;
+
+fn main() {
+    let args = env::args_os().skip(1).collect::<Vec<_>>();
+
+    let rustc = env::var_os("RUSTC_REAL").unwrap();
+    let mut cmd = Command::new(&rustc);
+    let any_time_passes = args.iter().any(|a| a.to_str() == Some("time-passes"));
+    if env::var_os("USE_PERF").is_some() && any_time_passes {
+        cmd = Command::new("perf");
+        cmd.arg("stat")
+            .arg("-x,")
```

Comment:
The manpage for perf stat specifically recommends not using `,` as the records apparently aren't quite CSV then -- e.g., `,` isn't escaped or put inside quotes. I don't think it matters for us though...